test/test_tensor.py

```diff
@@ -303,6 +303,12 @@ def test_copy_from_numpy_unaligned(self):
     # force device copy - to() is opt'd away - Tensor(dev)/1 is ignored
     np.testing.assert_allclose(ua_arr, (Tensor(ua_arr)/Tensor(1)).numpy())
 
+  def test_item(self):
+    assert Tensor(10).item() == 10
```

Comment:
cool! can you test the output dtype too? like bool to bool, int to int and float to float